apps/cowswap-frontend/src/modules/account/containers/AccountDetails/index.tsx

```diff
@@ -66,11 +66,6 @@ import { SurplusCard } from './SurplusCard'
 import { useDisconnectWallet } from '../../hooks/useDisconnectWallet'
 import { CreationDateText } from '../Transaction/styled'
 
-export const NETWORK_LABELS: { [chainId in ChainId]?: string } = {
```

Comment:
Why didn't we have label for Mainnet?

Reply:
I guess because it didn't expect to have invalid network error for mainnet...